bitnami/grafana-mimir/templates/_helpers.tpl

```diff
@@ -0,0 +1,372 @@
+{{- printf "%s-%s" (include "common.names.fullname" .) "distributor" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Return the proper Grafana Mimir ingester fullname
+*/}}
+{{- define "grafana-mimir.ingester.fullname" -}}
+{{- printf "%s-%s" (include "common.names.fullname" .) "ingester" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Return the proper Grafana Mimir querier fullname
+*/}}
+{{- define "grafana-mimir.querier.fullname" -}}
+{{- printf "%s-%s" (include "common.names.fullname" .) "querier" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Return the proper Grafana Mimir query-frontend fullname
+*/}}
+{{- define "grafana-mimir.query-frontend.fullname" -}}
+{{- printf "%s-%s" (include "common.names.fullname" .) "query-frontend" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Return the proper Grafana Mimir store-gateway fullname
+*/}}
+{{- define "grafana-mimir.store-gateway.fullname" -}}
+{{- printf "%s-%s" (include "common.names.fullname" .) "store-gateway" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Return the proper Grafana Mimir gateway fullname
+*/}}
+{{- define "grafana-mimir.gateway.fullname" -}}
+{{- printf "%s-%s" (include "common.names.fullname" .) "gateway" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Return the proper Grafana Mimir ruler fullname
+*/}}
+{{- define "grafana-mimir.ruler.fullname" -}}
+{{- printf "%s-%s" (include "common.names.fullname" .) "ruler" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Return the proper Grafana Mimir ruler fullname
+*/}}
+{{- define "grafana-mimir.gossip-ring.fullname" -}}
+{{- printf "%s-%s" (include "common.names.fullname" .) "gossip-ring" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Return the proper Grafana Mimir alertmanager fullname
+*/}}
+{{- define "grafana-mimir.alertmanager.fullname" -}}
+{{- printf "%s-%s" (include "common.names.fullname" .) "alertmanager" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Return the proper Grafana Mimir overrides-exporter fullname
+*/}}
+{{- define "grafana-mimir.overrides-exporter.fullname" -}}
+{{- printf "%s-%s" (include "common.names.fullname" .) "overrides-exporter" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Return the proper Grafana Mimir query-scheduler fullname
+*/}}
+{{- define "grafana-mimir.query-scheduler.fullname" -}}
+{{- printf "%s-%s" (include "common.names.fullname" .) "query-scheduler" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Alertmanager http prefix
+*/}}
+{{- define "grafana-mimir.alertmanager.httpPrefix" -}}
+{{- print "/alertmanager" -}}
+{{- end -}}
+
+
+{{/*
+Prometheus http prefix
+*/}}
+{{- define "grafana-mimir.prometheus.httpPrefix" -}}
+{{- print "/prometheus" -}}
+{{- end -}}
+
+
+{{/*
+Get the Grafana Mimir configuration configmap.
+*/}}
+{{- define "grafana-mimir.mimir.configmapName" -}}
+{{- if .Values.mimir.existingConfigmap -}}
+    {{- .Values.mimir.existingConfigmap -}}
```

Comment:
Based on my personal experience with Supabase and the Kong subchart, we may want to allow templating on this